pipeline/validation-build.yaml

```diff
@@ -44,3 +44,25 @@ jobs:
 
       - script: node $(system.defaultWorkingDirectory)/pipeline/verify-clean-lockfile.js
         displayName: verify that building did not change the lockfile unexpectedly
+
+      - task: PublishPipelineArtifact@1
+        displayName: publish artifact
+        inputs:
+          targetPath: '$(system.defaultWorkingDirectory)/AccessibilityInsightsForAndroidService/app/build/outputs/apk/debug/app-debug.apk'
+          artifactName: debugApk
+
+  - template: ./e2e-emulator-template.yaml
```

Comment:
It might be less redundant to use a `job` with a [strategy](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/jobs-job-strategy?view=azure-pipelines#strategy-parallel) instead of repeating the same template 4 times

Reply:
Good call!